corehq/apps/reports/filters/location.py

```diff
@@ -0,0 +1,63 @@
+from dimagi.utils.decorators.memoized import memoized
+from corehq.apps.locations.models import SQLLocation
+from .users import ExpandedMobileWorkerFilter
+from .api import EmwfOptionsView
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext_lazy
+from corehq import toggles
+
+
+class LocationGroupFilter(ExpandedMobileWorkerFilter):
```

Comment:
may be worth adding a doc string explaining what this is and how it's different from `ExpandedMobileWorkerFilter`. asking mainly because reviewing this it wasn't totally clear to me without digging through the code (I think you're explicitly removing certain types of data from the filter?)